mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -524,18 +548,26 @@ def create_model_version(
         headers, run = self._get_run_and_headers(run_id)
         source_workspace_id = self._get_workspace_id(headers)
         notebook_id = self._get_notebook_id(run)
+        table_detail_list = self._get_table_details(run)
         job_id = self._get_job_id(run)
         job_run_id = self._get_job_run_id(run)
         extra_headers = None
         if notebook_id is not None or job_id is not None:
             entity_list = []
+            securable_list = []
             if notebook_id is not None:
                 notebook_entity = Notebook(id=str(notebook_id))
                 entity_list.append(Entity(notebook=notebook_entity))
             if job_id is not None:
                 job_entity = Job(id=job_id, job_run_id=job_run_id)
                 entity_list.append(Entity(job=job_entity))
-            lineage_header_info = LineageHeaderInfo(entities=entity_list)
+            if table_detail_list is not None:
+                for table_name, table_id in table_detail_list:
+                    table_entity = Table(name=table_name, id=table_id)
```

Comment:
any reason why the `table_entity` is constructed here rather than in  `_get_table_details()`? that way we can avoid packing/unpacking this array of tuples.